code/__defines/subsystem-priority.dm

```diff
@@ -3,11 +3,12 @@
 // SS_BACKGROUND handles high server load differently than Normal and SS_TICKER do.
 // Higher priority also means a larger share of a given tick before sleep checks.
 
-#define SS_PRIORITY_DEFAULT 50          // Default priority for all processes levels
+#define SS_PRIORITY_DEFAULT 50         // Default priority for all processes levels
 
 // SS_TICKER
+#define SS_PRIORITY_OVERLAY        200 // Applies overlays. May cause overlay pop-in if it gets behind.
```

Comment:
I believe this is an unreasonably large number.

Reply:
O7 sets this to 100.